miner/worker.go

```diff
@@ -1127,9 +1130,9 @@ func (w *worker) commitRip7560TransactionsBundle(env *environment, txs *types.Ex
 	}
 	receipts = append([]*types.Receipt{receipt}, receipts...)
 
+	env.tcount += len(env.txs) + len(validatedTxs)
```

Comment:
Why did you revise this? Won't be `env.txs` counted duplicated then?

Reply:
Yes, this is unnecessary. I'll remove it